core/src/test/java/org/elasticsearch/search/internal/InternalSearchHitTests.java

```diff
@@ -94,4 +215,45 @@ public void testHasSource() {
         searchHit.sourceRef(new BytesArray("{}"));
         assertTrue(searchHit.hasSource());
     }
+
+    public void testParseShardId() {
```

Comment:
then create ShardIdTests and add this test to it, which is what we have. at least what we are adding is consistent then